build.gradle

```diff
@@ -76,19 +70,6 @@ subprojects {
         from sourceSets.main.output
     }
 
-    if (!version.endsWith("SNAPSHOT")) {
```

Comment:
We will not be having snapshots anymore.

Reply:
@jpkrohling what do you mean by not having snapshots? Is a not released version going to compile as a.b.c ?

Reply:
Is it skipKit limitation or just the way how it's done here? 

I would like to keep snapshot builds. If we go away form snapshots we will have to remove version from local repo to include released changes. It's not very convenient.

Reply:
I don't want to publish snapshot to public repo. I just want -snapshot to be used for any not released version.

Reply:
I want snapshots to build and installed to my local repo. I use it often when working on different components which include jaeger.

Reply:
> I use it often when working on different components which include jaeger.

I considered this case as well, but in the end, I *think* my workflow will barely change. I can always update the `version.properties` to include `-SNAPSHOT` in the version string.

The advantage of dropping snapshots altogether is that there's no logic needed to handle this scenario during the releases, so, we need only one version change, instead of two (0.0.1-SNAPSHOT -> 0.0.1 -> 0.0.2-SNAPSHOT). 

A positive side-effect, IMO, is that I can test the real version string on PRs that I prepare before the release is done, like I did for the SmallRye PR: https://github.com/smallrye/smallrye-opentracing/pull/25 . The branch was tested and ready to be sent as PR before the release reached Maven Central.

Reply:
The snapshot is there for some reasons. For instance avoid using local changes where you expect released version. If it's dropped it's hard to know what version of client is being used in your local development.  

> A positive side-effect, IMO, is that I can test the real version string on PRs that I prepare before the release is done, like I did for the SmallRye PR: smallrye/smallrye-opentracing#25 . The branch was tested and ready to be sent as PR before the release reached Maven Central.

Or you could have done changes which you forgot that you made and the change to smallrye would not work bc you expect something else in the repo :)

Reply:
How about we add `publishToMavenLocal.enabled = false` to `build.gradle` and leave snapshots out of this PR, at least at the beginning? Our use case doesn't apply to everyone and it doesn't happen _everyday_, so, I'd rather have the release script and the git history simplified.  Then, when we need to consume a non-published artifact (snapshot, in maven terms), we change the local `version.properties` to include `-SNAPSHOT` and manually call `./gradlew publishToMavenLocal` when needed.

If it turns out that we can't live without snapshots (like everything other than maven can), than we reintroduce it in the release script, as an additional commit.